documentation/release_notes.markdown

```diff
@@ -8,85 +8,66 @@ canonical: "/puppetserver/latest/release_notes.html"
 
-Server 2.4.0 expands the syntax for [`allow` and `deny` parameters](./config_file_auth.markdown#allow-allow-unauthenticated-and-deny) in Server's `auth.conf` rules to allow for a map of `extensions` to match.
+-   [SERVER-1470](https://tickets.puppetlabs.com/browse/SERVER-1470)
+-   [SERVER-1247](https://tickets.puppetlabs.com/browse/SERVER-1247)
+-   [SERVER-1213](https://tickets.puppetlabs.com/browse/SERVER-1213)
 
--   [TK-293: tk-auth should support x.509 extensions for authentication instead of just certname](https://tickets.puppetlabs.com/browse/TK-293)
+### Bug fix: Unrecognized parse-opts
 
-Server 2.4.0 also reads custom OID shortname maps defined in Puppet's [`custom_trusted_oid_mapping.yaml`](https://docs.puppet.com/puppet/latest/reference/config_file_oid_map.html).
+Puppet Server 2.4.x used a deprecated API for a Clojure CLI option-parsing library. As a result, calls to `puppetserver gem` (either directly, or indirectly by using a `puppetserver_gem` package resource) generated unexpected warning messages:
 
--   [SERVER-1150](https://tickets.puppetlabs.com/browse/SERVER-1150)
--   [SERVER-1245](https://tickets.puppetlabs.com/browse/SERVER-1245)
+    Warning: Could not match Warning: The following options to parse-opts are unrecognized: :flag
 
-### New feature: Integrate with systemd services on Debian and Ubuntu
+Puppet Server 2.5.0 updates this library, which prevents this error message from appearing.
 
-Puppet Server 2.4.0 adds integration with `systemd` on Debian 8 and newer, and Ubuntu 16.04 LTS.
+-   [SERVER-1378](https://tickets.puppetlabs.com/browse/SERVER-1378)
 
--   [EZ-48](https://tickets.puppetlabs.com/browse/EZ-48)
+### Bug fix: Puppet Server no longer ships with an empty PID file
 
-### Improvement: Responses to unauthenticated HTTPS requests include less information
+When installed on CentOS 6, Puppet Server 2.4.x included an empty PID file. When running `service puppetserver status`, Puppet Server returned an unexpected error message: `puppetserver dead but pid file exists`.
 
-When responding to unauthorized HTTPS requests, previous versions of Puppet Server 2.x returned the requester's IP address and [authorization rule][auth.conf] in addition to logging the failed request. Puppet Server 2.4.0 removes this information from the response and directs the responder to consult the server logs for details.
+When performing a clean installation of Puppet Server 2.5.0, no PID file is created, and `service puppetserver status` should return the expected `not running` message.
 
--   [TK-360: Error messages returned to client should not include IP or rule blocking the request](https://tickets.puppetlabs.com/browse/TK-360)
+-   [SERVER-1455](https://tickets.puppetlabs.com/browse/SERVER-1455)
+-   [EZ-84](https://tickets.puppetlabs.com/browse/EZ-84)
 
-### New feature: `always_retry_plugins` setting to configure Puppet feature caching
+### New feature: Signing CSRs with OIDs from a new arc
```

Comment:
My main concern about bumping the bootstrap stuff from the top is because it represents a potential break on upgrade.